README.md

```diff
@@ -90,20 +90,45 @@ vars:
 ```
 
 ### Pivoting out Metadata Properties
-By default, this package selects the `metadata` JSON field within the `customer`, `charge`, `invoice`, `payment_intent`, `payment_method`, `payout`, `plan`, `refund`, and `subscription` source tables. However, you likely have properties within the `metadata` JSON field you would like to pivot out and include in the respective downstream staging model.
+Oftentimes you may have custom fields within your source tables that is stored as a JSON object that you wish to pass through. By leveraging the `metadata` variable, this package pivot out fields into their own columns. The metadata variables accept dictionaries in addition to strings.
+
+Additionally, if you happen to be using a reserved word as a metadata field, any otherwise incompatible name, or just wish to rename your field, Below are examples of how you would add the respective fields.
+
+The `metadata` JSON field is present within the `customer`, `charge`, `invoice`, `payment_intent`, `payment_method`, `payout`, `plan`, `refund`, and `subscription` source tables. To pivot these fields out and include in the respective downstream staging model, add the respective variable(s) to your root `dbt_project.yml` file like below.
+
+```yml
+vars: 
+  stripe__charge_metadata:
+    - metadata_field_1
+  stripe__invoice_metadata: 
+    - metadata_field_2
+  stripe__payment_intent_metadata:
+    - name: incompatible.field
+      alias: rename_incompatible_field
+  stripe__payment_method_metadata:
+    - name: field_is_reserved_word
+      alias: field_is_reserved_word_xyz
+  stripe__payout_metadata:
+    - name: 123
+      alias: one_two_three
+  stripe__plan_metadata:
+    - name: rename
+    - alias: renamed_field
+  stripe__refund_metadata:
+    - metadata_field_3
+    - metadata_field_4
+  stripe__subscription_metadata:
+    - metadata_field_5
+  stripe__customer_metadata:
+    - metadata_field_6
```

Comment:
Hey @fivetran-reneeli thanks for making this update. The condensed version looks great. One update I would like to suggest is keeping the declaration of the variables consistent in our documentation. While we are able to allow customers to include the fields without the `name` or `alias`, I do want to instruct them to use it. I feel we open ourselves for more confusion if we don't.
```suggestion
vars: 
  stripe__charge_metadata:
    - name: metadata_field_1
  stripe__invoice_metadata: 
    - name: metadata_field_2
  stripe__payment_intent_metadata:
    - name: incompatible.field
      alias: rename_incompatible_field
  stripe__payment_method_metadata:
    - name: field_is_reserved_word
      alias: field_is_reserved_word_xyz
  stripe__payout_metadata:
    - name: 123
      alias: one_two_three
  stripe__plan_metadata:
    - name: rename
    - alias: renamed_field
  stripe__refund_metadata:
    - name: metadata_field_3
    - name: metadata_field_4
  stripe__subscription_metadata:
    - name: metadata_field_5
  stripe__customer_metadata:
    - name: metadata_field_6
```